src/core/service/token/authutils.go

```diff
@@ -42,6 +42,12 @@ func init() {
 	privateKey = config.TokenPrivateKeyPath()
 }
 
+// ClaimSet ...
+type ClaimSet struct {
+	token.ClaimSet
+	PolicyCheck bool `json:"policy_check"`
```

Comment:
I don't think we should add `policy check` into the token, though we add this field to bypass the scan.
I think the token needs re-design, please discuss with @steven-zou I recall he mentioned there should be a `scan` in the claim, that makes more sense, because that means the token is for scan the image.

Reply:
+ @steven-zou please add comments

Reply:
updated